### DISCUSSION TITLE
Standup (Thursday, 27th)

### DISCUSSION THREAD

**Discussion by antonieto** [BBGTC/food-bank-app#24] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Not much, basic styling

**What are your priorities for the next 24 hours?**:
 - Keep working on it

**Blockers**:
 - Semana Tec

**Shoutouts**:
 - @antonieto @BrandonMagana for their work

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Merged #17 and #18 
 - Protected develop

**What are your priorities for the next 24 hours?**:
 - Read about database seeding
 - Read about Django channels
 - Take on another issue
 - Set up testing CI

**Blockers**:
 - None

**Shoutouts**:
 - @DavBot02 and @BrandonMagana for his code review

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Keep reviewing react

**What are your priorities for the next 24 hours?**:
 -  Help @antonieto  with github actions

**Blockers**:
 - Semana twerk

**Shoutouts**:
 - @antonieto  for keep working hard
